src/Components/SinglePost-Sidebar/SinglePostSidebarCard.tsx

```diff
@@ -1,16 +1,29 @@
 import React from "react";
-import "../../styles.scss";
+import { Link } from "react-router-dom";
+import "./singlePostSidebarCard.scss";
+import { useSelector } from "react-redux";
 import NavigationBar from "../CommonComponents/NavigationBar";
 import Sidebar from "./Sidebar";
 import prevpostimg from "../../assests/images/prevpost.png";
 import nextpostimg from "../../assests/images/nextpost.png";
-import CommentCard from "./CommentCard";
-import store from "../BlogOneColumn/foodzeroStore";
+import CommentCard from "../CommonComponents/CommentCard";
+import AuthorProfileCard from "../CommonComponents/AuthorProfileCard";
+import Tags from "../CommonComponents/Tags";
+import { Postinfo } from "../../Store/blogContent";
 
 
-export default function SinglePostSidebarCard({ obj }: any) {
+type sidebarPostType = {
+    obj: Postinfo
+}
 
-    const state = store.getState();
+export default function SinglePostSidebarCard({ obj }: sidebarPostType) {
+
+    const AuthorData: any = useSelector((state: any) => state.blogData);
+    const commentData: any = useSelector((state: any) => state.commentData);
+
+    const scrollToTop = () => {
+        window.scrollTo(0, 0)
+    }
```

Comment:
could have used the method from common helper file

Reply:
updated these lines with common method